lib/plausible_web/plugs/auto_auth_plug.ex

```diff
@@ -10,6 +10,7 @@ defmodule PlausibleWeb.AutoAuthPlug do
     cond do
       Keyword.fetch!(Application.get_env(:plausible, :selfhost), :disable_authentication) ->
         conn
+        |> Phoenix.Controller.put_view(AuthView)
```

Comment:
500 error is fixed by manually replacing the view responsible for rendering the templates. Since the plug is called from page_controller, the view was page_view which didn't have the login_form.html template.